Filter/Field/RangeFilterField.php

```diff
@@ -0,0 +1,99 @@
+<?php
+// https://www.elastic.co/guide/en/elasticsearch/client/php-api/current/_search_operations.html
+namespace Markup\NeedleBundle\Filter\Field;
+
+class RangeFilterField
+{
+    /**
+     * @var string
+     */
+    private $fieldName;
+
+    /**
+     * @var string|null
+     */
+    private $minValue;
+
+    /**
+     * @var string|null
+     */
+    private $maxValue;
+
+    public function __construct(string $fieldName, array $value)
```

Comment:
`$value` is too generic.. I have no clue how to construct this object without looking at it.. this should likely be `\DateTime $start, \DateTime $end` surely?